config/backup.php

```diff
@@ -60,7 +60,7 @@
                 'mysql',
             ],
         ],
-
+        'mongodb_user_auth' => env('MONGO_DB_AUTH', 'admin'),
```

Comment:
I'm thinking we should add a new config value, but use the `dump` config key on a database connection. Example here: https://docs.spatie.be/laravel-backup/v6/installation-and-setup#dumping-the-database

Reply:
I don't see a reason why `mongodb_user_auth` should be in `config/backup.php` and can't live as extra option in the dump config like mentioned here: https://docs.spatie.be/laravel-backup/v6/installation-and-setup#dumping-the-database

Reply:
I fix what you ask @freekmurze. I have also added some test with mongo, but how can i fake a mongodb connection like you do in function [`createSQLiteDatabase`](https://github.com/spatie/laravel-backup/blob/25571aeb16ba8579b8426c96a9c4134394502be3/tests/TestCase.php#L159)?